ftw/colorbox/profiles/default/registry.xml

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0"?>
 <registry>
-    <records interface="ftw.colorbox.interfaces.IColorboxSettings" />
+  <records interface="ftw.colorbox.interfaces.IColorboxSettings" />
+
+  <records interface="Products.CMFPlone.interfaces.controlpanel.IImagingSchema"
+           prefix="plone">
+    <value key="allowed_sizes" purge="false">
+      <element>colorbox 2000:2000</element>
+    </value>
+  </records>
+
+    <records prefix="plone.bundles/ftw-colorbox-resources"
```

Comment:
This shouldn't be in a plone4 profile, should it?

Reply:
I think I comitted unrelated changes. Let me sort this out.

Reply:
Fixed 👍